### PR TITLE
add support for Ubuntu 20.04

### PR DESCRIPTION
**Pull request by Octav14n**
I added the instructions and changes I needed to make to make the program runable on Ubuntu 20.04.

g++ seems to need the input-file and output-parameter before the includes are declared (why ever that is?!)
Ubuntu 20.04 does not have the needed packages in its repository so I added a section in README on how to install the dependencies.

### CODE REVIEW

File: tools/list_keyrings.sh [hunk 1]
```diff
@@ -1,4 +1,5 @@
 #!/bin/bash
-eval g++ $(pkg-config --cflags --libs gnome-keyring-1) list_keyrings.cc -o list.out &&
+cd $(dirname $0)
```
Comment (recolic) [inline tools/list_keyrings.sh]: Please write `cd $(dirname $0) &&`. Any small error should fail the whole script, as early as possible.

File: README.md [hunk 2]
```diff
@@ -19,7 +19,20 @@ I encrypt the `keyring-name : password` pair with GnuPG and save it as `secret-f
 
 > I recommend you to **configure Yubikey as GPG smartcard**. The system would just ask you to unlock gnome-keyring with your default GPG software. You may generate a new GPG key for yubikey, or move your existing GPG key into yubikey. Refer to google for these knowledge. 
 
-First, build the project from source.
+First, install packages.
```
Comment (recolic) [inline README.md]: Not all users are using ubuntu. Please Add a new section named `dependency`, and add an entry `ubuntu 20.04`. Archlinux/Gentoo/fedora/opensuse users may use different packages.

**Comment by Octav14n** [comment]
OK, I made the suggested changes.
I found a line I accidentally deleted (I have no idea how/why) and re-inserted it too.

**Review by recolic — APPROVED** [review]

**Comment by recolic** [comment]
Thanks!

**Pull request merged**